src/sentry/api/endpoints/organization_events_stats.py

```diff
@@ -35,6 +35,7 @@ def get(self, request, organization):
         snuba_args = self.get_field(request, snuba_args)
 
         result = snuba.transform_aliases_and_query(
+            dataset="events",
```

Comment:
This won't always be true soon. In #14857 I plan on adding code to detect the dataset based on the end user query contents. This and the changes to organization_events_distribution will be in conflict.

Reply:
Hmmm, I see. Would it make more sense to make the default dataset = "events" (so [these two commits](https://github.com/getsentry/sentry/pull/14909/files/3d9cbee85dcdefc50365c3139992b163120ad45f)) while we're both working on this side of the codebase?